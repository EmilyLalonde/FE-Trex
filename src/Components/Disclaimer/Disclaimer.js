import React from 'react';
import {
  StyleSheet,
  Text,
  View,
  ScrollView,
  TouchableOpacity
} from "react-native";

const Disclaimer = () => {
  return (
    <View>
    <Text>This application is designed for your safety. Please do not add followers that you do not know of trust!</Text>
    <TouchableOpacity>Continue</TouchableOpacity>
    </View>
  )
}

export default Disclaimer