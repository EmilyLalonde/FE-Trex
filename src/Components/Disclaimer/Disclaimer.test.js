import React from "react";
import "react-native";
import Disclaimer from "./Disclaimer";
import { shallow } from "enzyme";

describe("Disclaimer", () => {
  it("should return true", () => {
    expect(true).toEqual(true);
  });
  it("should match the snapshot when loading", () => {
    let wrapper = shallow(<Disclaimer navigation={{ navigate: jest.fn() }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
