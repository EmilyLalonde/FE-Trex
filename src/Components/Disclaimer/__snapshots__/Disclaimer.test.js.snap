// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Disclaimer should match the snapshot when loading 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
        "marginVertical": 40,
        "textAlign": "center",
        "width": "auto",
      }
    }
  >
    This application is designed for your safety.  Please do not add followers  that you do not know or trust!
  </Text>
  <TouchableOpacity
    activeOpacity={0.2}
  >
    <Text
      style={
        Object {
          "backgroundColor": "#1C4263",
          "borderColor": "white",
          "borderRadius": 8,
          "borderStyle": "solid",
          "borderWidth": 1,
          "color": "white",
          "fontSize": 30,
          "height": 60,
          "margin": 20,
          "padding": 10,
          "textAlign": "center",
          "width": "auto",
        }
      }
    >
      Continue
    </Text>
  </TouchableOpacity>
</View>
`;
